script/release-workflow/docker-push.sh

```diff
@@ -1,15 +1,29 @@
 #!/bin/sh
 
-set -euo >/dev/null
+set -eo >/dev/null
 
-if [ -n "${MAJOR_TAG}" ]; then
-  docker tag ${DOCKER_IMAGE_ORG_AND_NAME}:latest ${DOCKER_IMAGE_ORG_AND_NAME}:${MAJOR_TAG}
-  docker push ${DOCKER_IMAGE_ORG_AND_NAME}:${MAJOR_TAG}
+
+if [ -z "${TAG}" ]; then
+  echo TAG env var not set
+  exit 1
 fi
 
-docker tag ${DOCKER_IMAGE_ORG_AND_NAME}:latest ${DOCKER_IMAGE_ORG_AND_NAME}:${TAG}
-docker push ${DOCKER_IMAGE_ORG_AND_NAME}:${TAG}
+## These will use cached builds, so wont build every time.
+
+if [ -n "${TAG}" ]; then
+  docker buildx build --platform=linux/amd64,linux/arm64,linux/arm \
+    --output=type=image,push=true \
+    -t ${DOCKER_IMAGE_ORG_AND_NAME}:${TAG} .
+fi
```

Comment:
this creates a multi-manifest build `--platform=linux/amd64,linux/arm64,linux/arm`

and will push to the registry as well `--output=type=image,push=true`

They re-use the buildx cache populated in `docker-build.sh`